unison-src/transcripts-manual/remote-tab-completion.output.md

```diff
@@ -3,8 +3,6 @@
 Note: this makes a network call to share to get completions
 
 ``` ucm
-.> debug.tab-complete pull unison.pub
-
-   unison.public
```

Comment:
I guess this is an old change in a transcript that's run infrequently, but what happened here?

Reply:
I'm actually not sure, it doesn't look related to these changes. I think I manually ran more transcripts in `transcripts-manual` than strictly necessary, just trying to hit all the random stuff that calls `add`... could revert or leave it, what do you think?

Reply:
I'll take another look to see if this whole transcript is deprecated or what.  remote tab completion sounds like a "loose code" thing.